RecoEgamma/EgammaPhotonProducers/python/allConversions_cfi.py

```diff
@@ -64,5 +65,3 @@
 
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 phase2_hgcal.toModify( allConversions, bypassPreselGsf = cms.bool(False) )
-from Configuration.Eras.Modifier_fastSim_cff import fastSim
```

Comment:
Shouldn't this be restored?

Reply:
I will look carefully in this stuff. It may be yes, that some I developed some configs on an old base.